library/standard_tables_capture.inc

```diff
@@ -11,12 +11,16 @@
  * @author    Brady Miller <brady.g.miller@gmail.com>
  * @author    Roberto Vasquez <robertogagliotta@gmail.com>
  * @author    Stephen Waite <stephen.waite@cmsvt.com>
+ * @author    Robert Down <robertdown@live.com>
  * @copyright Copyright (c) 2011 Phyaura, LLC <info@phyaura.com>
  * @copyright Copyright (c) 2019 Brady Miller <brady.g.miller@gmail.com>
  * @copyright Copyright (c) 2019 Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2021 Robert Down <robertdown@live.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+ use PhpOffice\PhpSpreadsheet\Reader\Xlsx;
```

Comment:
wooohoooo, i think this is the first use of PhpOffice that will get into core

![2DV](https://user-images.githubusercontent.com/278968/145728236-e66eea2c-1669-4eb3-8da7-34880a5e64df.gif)